src/lib/constants.ts

```diff
@@ -24,6 +24,29 @@ export const FORMATS: ColorFormatId[] = [
   'srgb',
 ];
 
+export type FormatGroup = {
+  name: string;
+  formats: ColorFormatId[];
+  gamutFormat?: ColorFormatId;
+  gamutName?: string;
+};
+
+export const FORMAT_GROUPS: FormatGroup[] = [
+  {
+    name: 'sRGB Formats',
+    formats: ['hex', 'hsl', 'srgb'],
+    gamutFormat: 'srgb',
+    gamutName: 'sRGB',
+  },
+  { name: 'Unbounded Spaces', formats: ['lab', 'lch', 'oklab', 'oklch'] },
+  {
+    name: 'Display p3 Space',
+    formats: ['p3'],
+    gamutFormat: 'p3',
+    gamutName: 'p3',
```

Comment:
I think elsewhere we're capitalizing `P3` for display?

```suggestion
    name: 'Display P3 Space',
    formats: ['p3'],
    gamutFormat: 'p3',
    gamutName: 'P3',
```